src/main/java/org/embl/mobie/io/ome/zarr/util/N5OMEZarrCacheArrayLoader.java

```diff
@@ -64,61 +66,27 @@ public A loadArray(final long[] gridPosition) throws IOException {
         }
 
         if (block == null) {
-            return (A) zarrArrayCreator.createEmptyArray(gridPosition);
+            return (A) zarrArrayCreator.createEmptyArray( gridPosition );
```

Comment:
I think you could remove the spaces. I probably added them because it is my coding style.